Xpresso/model-template.yaml

```diff
@@ -7,13 +7,8 @@ args:
         #md5: 9d00a3bc614da81655328b6e278569e2
 
 default_dataloader:
-    defined_as: kipoiseq.dataloaders.SeqIntervalDl
-    default_args:
-        auto_resize_len: 10500
-        alphabet_axis: 1
-        alphabet: "ACGT" # Order of letters in 1-hot encoding
-        use_strand: False
-        ignore_targets: False # if True, dont return any target variables
+    defined_as: kipoiseq.dataloaders.AnchoredGTFDl
+
```

Comment:
Please add default_args for num_upstream, num_downstream. Probably need to fix `override_default_kwargs` https://github.com/kipoi/kipoi/blob/c1694eb5a0f7ad4607edcf389dcc927a75cf15b4/kipoi/specs.py#L816 to pin args that don't have a default value yet.

Reply:
I have pinned the values in the yaml docstring [here](https://github.com/kipoi/kipoiseq/blob/master/kipoiseq/dataloaders/sequence.py#L407-L412).
The bug will be fixed here - https://github.com/kipoi/kipoi-utils/issues/28